src/pages/drawing/drawing.vue

```diff
@@ -2,7 +2,11 @@
   <div>
     <h2>{{ title }}</h2>
     <p>
-      To start drawing send <InlineCode>StartDrawingRequest</InlineCode>.<br>
+      With drawing features of Oskari RPC API you can give end-user possibility to make their own markings to the map.
+      <br><br>
+      In order to save user made markings they should be saved as features of the map. These features can then be programmatically listened for events and used to trigger different functions of application for example.
```

Comment:
```suggestion
      The drawn features trigger events that can be programmatically listened to and used to trigger different functions in the application that is using an Oskari-based embedded map.
```
And somewhere near the end of the page:
```
If you want to store the markings/drawings the user made on the map after they have been drawn it's best to add them as features to the map with AddFeaturesToMapRequest based on the GeoJSON from DrawingEvent. Otherwise stopping the draw mode or starting a new drawing can accidentally erase the previous markings from the map. It's ok to try and keep the markings on the drawing layer depending on your app, but most real world applications probably want more control over the markings. Adding them as features allow styling, click events and other benefits. 
```